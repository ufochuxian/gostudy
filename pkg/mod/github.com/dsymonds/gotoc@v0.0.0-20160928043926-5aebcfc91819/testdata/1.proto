// Just the basics

syntax = "proto2";

message A {
  required int32 a = 1;
  optional string b = 2;
  // skip field 3
  repeated sint64 c = 4;
}
