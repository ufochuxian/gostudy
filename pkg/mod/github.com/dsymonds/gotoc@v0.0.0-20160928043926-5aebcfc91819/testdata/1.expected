file: <
  name: "1.proto"
  message_type: <
    name: "A"
    field: <
      name: "a"
      number: 1
      label: LABEL_REQUIRED
      type: TYPE_INT32
    >
    field: <
      name: "b"
      number: 2
      label: LABEL_OPTIONAL
      type: TYPE_STRING
    >
    field: <
      name: "c"
      number: 4
      label: LABEL_REPEATED
      type: TYPE_SINT64
    >
  >
>
